### ISSUE TITLE
Invoke-IcingaCheckPartitionSpace - Output Unit

### ISSUE THREAD

**Issue by phil-or** [Icinga/icinga-powershell-plugins#259]
Hi Christian!

Thank you for your efforts in the further development of the Powershell framework!!!

While migrating to the CheckPartitionSpace I get a bit confused why the output in Icingaweb is always in GiB/MiB
![grafik](https://user-images.githubusercontent.com/36534023/146533914-86626d8c-89f1-4c4d-8725-52acd345ec93.png)

We usually use GB/MB for thresholds. What do you think about an additional argument where you can choose your "Output Unit"?

**Comment by LordHepipud** [comment]
Hello

Thank you for the issue. We updated the checks to properly use MiB, GiB and TiB, as this is the actual value displayed within the Windows Explorer on Windows.

While Windows states the free space in the explorer as `6.31GB`, they are actually GiB, which caused confusion in the past, while the values we displayed as GB didn't match the values the Windows Explorer reported.

I would not want to add another argument to convert the output units to something else, as these units are the proper result and do not cause different values.

For monitoring, you would simply have to write `-Warning 5GiB:` and the Framework will do the rest for you.
